core/Bench.carp

```diff
@@ -5,38 +5,51 @@
 
 (defmodule Bench
   (def min-runs 50)
+  (private min-runs)
```

Comment:
nitpick: move these private/hidden before the def to be consistent

Reply:
I cannot, since those are variables and the compiler will error otherwise. I’ll gladly do that when the compiler is fixed.

Reply:
Fair enough :)